lib/truffle/truffle/cext.rb

```diff
@@ -947,6 +947,10 @@ def rb_funcall_with_block(recv, meth, args, block)
     recv.public_send(meth, *args, &block)
   end
 
+  def rb_respond_to(object, name)
+    object.respond_to?(name)
```

Comment:
I think this is out of scope for this PR but I thought I'd mention that `rb_respond_to` may have slightly different behavior:
https://github.com/ruby/ruby/blob/650c45dcd70e791747c4b9084c100cb895e718cb/vm_method.c#L2036-L2050

than `obj_respond_to`:
https://github.com/ruby/ruby/blob/650c45dcd70e791747c4b9084c100cb895e718cb/vm_method.c#L2074